packages/mjml-navbar/src/Navbar.js

```diff
@@ -0,0 +1,181 @@
+import { BodyComponent } from 'mjml-core'
+import { merge } from 'lodash'
+import crypto from 'crypto'
+
+import conditionalTag, { msoConditionalTag } from 'mjml-core/lib/helpers/conditionalTag'
+
+const baseStyles = {
+  div: {
+    'width': '100%'
+  },
+  trigger: {
+    display: 'none',
+    'max-height': '0px',
+    'max-width': '0px',
+    'font-size': '0px',
+    overflow: 'hidden'
+  },
+  label: {
+    display: 'block',
+    cursor: 'pointer',
+    'mso-hide': 'all',
+    '-moz-user-select': 'none',
+    'user-select': 'none'
+  },
+  icoOpen: {
+    'mso-hide': 'all'
+  },
+  icoClose: {
+    display: 'none',
+    'mso-hide': 'all'
+  }
+}
+
+export default class MjNavbar extends BodyComponent {
+
+  static allowedAttributes = {
+    align: 'enum(left,center,right)',
+    'base-url': 'string',
+    'hamburger': 'string',
+    'ico-align': 'enum(left,center,right)',
+    'ico-open': 'string',
+    'ico-close': 'string',
+    'ico-color': 'color',
+    'ico-font-size': 'unit(px,%)',
+    'ico-font-family': 'string',
+    'ico-text-transform': 'string',
+    'ico-padding': 'unit(px,%)',
+    'ico-text-decoration': 'string',
+    'ico-line-height': 'unit(px,%)'
+  }
+
+  static defaultAttributes = {
+    'align': 'center',
+    'base-url': null,
+    'hamburger': null,
+    'ico-align': 'center',
+    'ico-open': '9776',
+    'ico-close': '8855',
+    'ico-color': '#000000',
+    'ico-font-size': '30px',
+    'ico-font-family': 'Ubuntu, Helvetica, Arial, sans-serif',
+    'ico-text-transform': 'uppercase',
+    'ico-padding': '10px',
+    'ico-text-decoration': 'none',
+    'ico-line-height': '30px'
+  }
+
+  static headStyle = breakpoint => `
+    noinput.mj-menu-checkbox { display:block!important; max-height:none!important; visibility:visible!important; }
+
+    @media only screen and (max-width:${breakpoint}) {
+      .mj-menu-checkbox[type="checkbox"] ~ .mj-inline-links { display:none!important; }
+      .mj-menu-checkbox[type="checkbox"]:checked ~ .mj-inline-links,
+      .mj-menu-checkbox[type="checkbox"] ~ .mj-menu-trigger { display:block!important; max-width:none!important; max-height:none!important; font-size:inherit!important; }
+      .mj-menu-checkbox[type="checkbox"] ~ .mj-inline-links > a { display:block!important; }
+      .mj-menu-checkbox[type="checkbox"]:checked ~ .mj-menu-trigger .mj-menu-icon-close { display:block!important; }
+      .mj-menu-checkbox[type="checkbox"]:checked ~ .mj-menu-trigger .mj-menu-icon-open { display:none!important; }
+    }
+  `
+
+  getStyles () {
+    const { mjAttribute, defaultUnit, getPadding } = this.props
+
+    return merge({}, baseStyles, {
```

Comment:
Can you now directly return the object instead of merging like here : https://github.com/mjmlio/mjml/blob/next/packages/mjml-social/src/SocialElement.js#L84-L113